modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,51 @@
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container. To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
+
+The public image based on Red Hat UBI has everything already set up for us in its https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile]. As an extra step, you need to “force” running the container as the “build” user since if we leave it to the default, it will be “randomed” and would not have the setup needed to run as non-root.
+
+You can directly edit or export to a specific namespace the buildah ClusterTask as described in the previous section. You can add this `securityContext`` to each build and push steps.
+
+[source,yaml,subs="attributes+"]
+----
+securityContext:
+    runAsUser: 1000
+----
+
+Because you are building as a local user, buildah saves its image cache to `/home/build/.local/cache`. Thus, instead of mounting the containers located in the `/var/lib/` directory, change the `mountPath` for the build to your local user container directory by changing the value for `mountPath` as follows:
```

Comment:
Starting here, the docs preview link in your PR does not render any of this content. See the screenshot: 
<img width="703" alt="Screen Shot 2022-05-31 at 1 52 31 PM" src="https://user-images.githubusercontent.com/70717303/171252790-cf8897a4-14bb-4edc-804d-4bcde31575fa.png">